common/app/views/fragments/inlineSvg.scala.html

```diff
@@ -1,4 +1,4 @@
-@(file: String, path: String = "", classes: Seq[String] = List())
-<span class="inline-@file @if(path){inline-@path} @classes.mkString(" ")">
+@(file: String, path: String = "", classes: Seq[String] = List(), attributes: Map[String, String] = Map())
```

Comment:
if we need more in future we can always refactor it again at that point.  The good thing about real parameters is you get all the compiler help, and we may never need to extend it